src/Sparrow/project.json

```diff
@@ -23,5 +23,7 @@
     }
   },
   "dependencies": {
+    "DotNetCross.Memory.Unsafe": "0.2.3",
```

Comment:
Until RC2 we needed it to be able to create a SIMD Vector from a pointer. This functionality is being included in RC2.